src/bootstrap/lib.rs

```diff
@@ -628,8 +628,10 @@ impl Build {
             self.run(&mut update(false));
         }
 
+        self.run(Command::new("git").args(&["stash", "save"]).current_dir(&absolute_path));
```

Comment:
Please consider using `git stash push` instead of `get stash save`. From the [manual](https://git-scm.com/docs/git-stash):
> save [-p|--patch] [-S|--staged] [-k|--[no-]keep-index] [-u|--include-untracked] [-a|--all] [-q|--quiet] [<message>]
This option is deprecated in favour of git stash push. It differs from "stash push" in that it cannot take pathspec. Instead, all non-option arguments are concatenated to form the stash message.